infra/modules/gcp-psoxy-bulk/variables.tf

```diff
@@ -18,7 +18,7 @@ variable "config_parameter_prefix" {
 variable "instance_id" {
   type        = string
   description = "id of psoxy instance"
-  default     = null
+  default     = ""
```

Comment:
Added it as empty; otherwise it fail later due a NPE doing the concat

Reply:
i'd prefer to solve that later by checking for `null`, just as it more clearly delineates the "unset"